README.md

```diff
@@ -80,6 +80,14 @@ We'll organise into different streams here. One kind request for motivation: Ple
 * Instagram Clone
 * Netflix Clone
 
+### PHP Development 
```

Comment:
Similar to this?
https://github.com/praveenscience/Hacktoberfest-Project-Ideas/pull/75/files